### PR TITLE
Paginate off-ice by month

### LINKED ISSUE #401: Off-ice booking view should be paginated by month instead of by week

### PR DESCRIPTION
**Pull request by fadwamahmoud**
Fixes #401 
Current behaviour (might be a bug): 
Both ice and off-ice views now show the same dates.
For `ice` the dates with both `ice` and `off-ice` slots are shown but only dates with `ice` slots actually display slots and vice versa for `off-ice`.

### CODE REVIEW

**Comment by silviot** [comment]
The ordering of dates is off:
Screenshot from 2022-03-13 13-07-06

**Comment by fadwamahmoud** [comment]
> The ordering of dates is off: slots are sorted by time of day, regardless of their date.
> 
> Screenshot from 2022-03-13 13-07-06

I didn't know whether to sort them or not because they weren't being sorted originally on off-ice view, they were just displayed as is from the selector. However, I added a function that sorts them by date.

**Review by silviot — APPROVED** [review]

File: eisbuk-admin/src/pages/customer_area/CustomerSlots/CustomerSlots.tsx [hunk 1]
```diff
@@ -59,22 +59,20 @@ const CustomerSlots: React.FC<Props> = ({
   const slotDates = Object.keys(slots);
 
   // if `view=book_ice` should order slot days mondays first and so on
-  // if `view=book_off_ice` display week in standard order
+  // if `view=book_off_ice` display in standard order
   const orderedDates =
-    view === CustomerRoute.BookIce ? orderByWeekDay(slotDates) : slotDates;
+    view === CustomerRoute.BookIce
+      ? orderByWeekDay(slotDates)
+      : orderByDate(slotDates);
```
Comment (silviot) [inline eisbuk-admin/src/pages/customer_area/CustomerSlots/CustomerSlots.tsx:66]: ISO dates sort lexycographically, so converting them to `DateTime` using the `fromISO` function is not needed (it's currently done in the `orderByDate` function). `slotDates.slice().sort()` should be enough

Comment (fadwamahmoud) [reply eisbuk-admin/src/pages/customer_area/CustomerSlots/CustomerSlots.tsx:66]: Why `slice()` though? @silviot

File: eisbuk-admin/cypress/integration/booking_spec.ts [hunk 2]
```diff
@@ -156,7 +156,10 @@ describe("Booking flow", () => {
       );
     });
 
-    it("disables only the locked bookings if locked/non-locked month overlap shown", () => {
+    /**
+     * @TODO make sure this test is necessary
+     */
+    it("enable booking in unlocked month and disable booking in locked month ", () => {
```
Comment (ikusteu) [inline eisbuk-admin/cypress/integration/booking_spec.ts:162]: You're right to ask yourself that...it isn't necessary

**Pull request merged**